src/test/fuzz.cpp

```diff
@@ -130,9 +136,12 @@ fuzz(std::string const& filename, el::Level logLevel,
 
 restart:
 {
+    srand(1);
+    gRandomEngine.seed(1);
+
     VirtualClock clock;
-    Application::pointer app1 = Application::create(clock, cfg1);
-    Application::pointer app2 = Application::create(clock, cfg2);
+    auto app1 = createTestApplication(clock, cfg1);
```

Comment:
why is app creation in the `main loop` instead of being before `restart`? Question in general, maybe it's the right thing to do for overlay; I am asking about the requirements for tests executed per message (in the `while` loop) vs per "session" (for lack of better term) inside the `restart` block.

Reply:
Moved setup before persistence loop and use LLVM mode allowing me to utilize ` while (__AFL_LOOP(PERSIST_MAX))`. Significant increase in exec/sec without compromising stability (determinism)